### PR TITLE
Run dgoss test in Github action

### PR DESCRIPTION
**Pull request by xendk**
Run tests in Github actions and remove Circle setup.

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          - "7.4"
+    steps:
+    - uses: actions/checkout@master
+    - uses: e1himself/goss-installation-action@v1.0.3
+    - name: Build image
+      run: |
+        docker build --file=Dockerfile-${{ matrix.php }} -t docker-drupal-php7-fpm:sut .
+    - name: Test image
+      run: |
+        GOSS_SLEEP=15 dgoss run -e GOSS_PHP_VERSION="${{ matrix.php }}" docker-drupal-php7-fpm:sut
```
Comment (kasperg) [inline .github/workflows/pull_request.yml:28]:
```suggestion
        # Sleep a bit to ensure that PHP-FPM has time to start up.
        GOSS_SLEEP=15 dgoss run -e GOSS_PHP_VERSION="${{ matrix.php }}" docker-drupal-php7-fpm:sut
```

I think the previous comment had some merrit in explaining why the sleep is needed.

**Review by arnested — APPROVED** [review]
LGTM :+1:

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          # transform "7.0" into "7".
+          - "5.6"
+          - "7.0"
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
+    steps:
+    - uses: actions/checkout@master
+    - uses: e1himself/goss-installation-action@v1.0.3
```
Comment (kasperg) [inline .github/workflows/pull_request.yml:21]:
```suggestion
    - uses: e1himself/goss-installation-action@v1
```

Perhaps just use the latest major version?

**Review by kasperg — APPROVED** [review]
:+1: LGTM

File: .github/workflows/pull_request.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+jobs:
+  goss_test:
+    name: Goss test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php:
+          # Strings, so they wont be parsed as floats which would
+          # transform "7.0" into "7".
+          - "5.6"
```
Comment (kasperg) [inline .github/workflows/pull_request.yml:13]: :+1: excellent commenting.

File: .github/workflows/pull_request.yml (hunk 2 above)
Comment (arnested) [reply .github/workflows/pull_request.yml:21]: if we extend the [dependabot](https://github.com/reload/docker-drupal-php7-fpm/blob/master/.github/dependabot.yml) with checking workflow files we should leave it to the patch level and then @dependabot will create pull requests

Comment (arnested) [reply .github/workflows/pull_request.yml:21]: See #46

Comment (xendk) [reply .github/workflows/pull_request.yml:21]: Besides, v1 seems not to be a branch tag: https://github.com/e1himself/goss-installation-action/tree/v1

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (xendk) [reply .github/workflows/pull_request.yml:28]: :+1:

**Pull request merged**